packages/components/root/src/root.css

```diff
@@ -152,3 +154,17 @@
 	-ms-grid-row-span: 1;
 	grid-row: 1 / 1;
 }
+
+:host(.ts-has-note) .content {
+	padding-top: var(--ts-unit-double);
+}
+
+:host ::slotted(ts-note) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the unknown "slotted" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aunknown-pseudo)